pages/getting-started/advanced-tutorial.html

```diff
@@ -679,8 +679,8 @@ <h2 class="page-header">Batch</h2>
 </div>
 <div class="col-md-3" style="float:right">
 
-    <nav class="tutorial-sidebar" aria-label="Table of contents">
-    <ul class="nav tutorial-sidenav" data-spy="affix">
+    <nav class="tutorial-sidebar" role="complementary" aria-label="Page Contents">
+    <ul class="nav tutorial-sidenav" role="navigation" data-spy="affix">
```

Comment:
```suggestion
    <ul class="nav tutorial-sidenav" data-spy="affix">
```
Nav implicitly will give the role of navigation.

Reply:
nit: Maybe consider having 
```html
    <aside class="tutorial-sidebar" aria-label="Page Contents">
```